include/drivers/sensor.h

```diff
@@ -80,7 +80,16 @@ enum sensor_channel {
 	/** Ambient temperature in degrees Celsius. */
 	SENSOR_CHAN_AMBIENT_TEMP,
 	/** Pressure in kilopascal. */
-	SENSOR_CHAN_PRESS,
+        SENSOR_CHAN_PRESS,
+        /** Euler heading(yawn) in degrees */
+        SENSOR_CHAN_EULER_H,
```

Comment:
I would prefer SENSOR_CHAN_EULER_Y
Moreover, it is "yaw" not "yawn".

Reply:
You can leave SENSOR_CHAN_EULER_H, but correct "yawn" into "yaw" in comment

Reply:
Are you still using [space] here?

Reply:
sorry now its tab, i messed that up in vi